tests/Common/ConfigurationRowTest.php

```diff
@@ -219,7 +226,7 @@ public function testConfigurationRowIsDisabledBooleanValue()
                     -H 'content-type: application/x-www-form-urlencoded' \
                     -H 'accept: */*' \
                     -H 'x-storageapi-token: " . STORAGE_API_TOKEN . "' \
-                    --data 'isDisabled=true&changeDescription=Row%20ABCD%20disabled' \
+                    --data 'isDisabled=false&changeDescription=Row%20ABCD%20enabled' \
```

Comment:
I don't like that previous test was changed. I think it would be better to add a new test `testCreateDisabledRow`.